Sources/TarkaUI/Extensions/Image+Extensions.swift

```diff
@@ -17,3 +17,18 @@ public extension Image {
 }
 
 
+public extension Image {
+  
+  init(icon: Icon) {
+    self.init(icon.rawValue, bundle: .module)
+  }
+}
+
+public enum Icon: String {
+  case checkBoxChecked = "checkbox_checked"
+  case checkBoxUnChecked = "checkbox_unchecked"
+  case toggleSwitchOn = "toggle_switch_on"
+  case toggleSwitchOff = "toggle_switch_off"
+  case toggleSwitchDisabledOff = "toggle_switch_diabled_off"
+  case toggleSwitchDisabledOn = "toggle_switch_diabled_on"
```

Comment:
why image assets are introduced..Are those not available in FluentIcons?

Reply:
yeah, it's custom one

Reply:
Did you check this with Yogesh?

Reply:
@gopinathcholan Updated toggleSwitch